stdlib/public/core/Integers.swift.gyb

```diff
@@ -809,69 +809,73 @@ def assignmentOperatorComment(operator, fixedWidth):
 def overflowOperationComment(operator):
     comments = {
         '+': """\
-  /// Returns the sum of this value and the given value along with a flag
-  /// indicating whether overflow occurred in the operation.
+  /// Returns the sum of this value and the given value, along with a Boolean
+  /// flag indicating whether overflow occurred in the operation.
```

Comment:
There's some vacillation here between "flag" and "Boolean value," probably a leftover from the days of `ArithmeticOverflow`. For consistency, I've standardized on "Boolean flag."

Reply:
Good catch on the "flag" business—these should all be "Boolean value indicating..." throughout to match the rest of the docs.